src/library/banshee/bansheefeature.cpp

```diff
@@ -127,6 +135,13 @@ void BansheeFeature::activateChild(const QModelIndex& index) {
     }
 }
 
+void BansheeFeature::onRightClickChild(const QPoint& globalPos, QModelIndex index) {
+    qDebug() << "";
+    qDebug() << "   BansheeFeature::onRightClickChild(" << m_pSidebarWidget << ")";
+    qDebug() << "       > BaseExternalLibraryFeature::onRightClickChild(...)";
```

Comment:
Can we comment out these qDebug() calls?

Reply:
jup, it's on the ToDo